Error in 'Manage' screen when using Oracle database for metl config

'limit' is not valid syntax in Oracle select statements
From log:
2017-11-20 15:05:09,016 ERROR [BackgroundRefresherService] [ui-refresher-1] Failed to execute sql: select id, agent_id, flow_id, deployment_id, deployment_name, agent_name, host_name, flow_name, status, start_time, end_time, create_by, last_update_by, parameters from METL_execution where status = ? order by create_time desc limit 100
org.jumpmind.db.sql.SqlException: Failed to execute sql: select id, agent_id, flow_id, deployment_id, deployment_name, agent_name, host_name, flow_name, status, start_time, end_time, create_by, last_update_by, parameters from METL_execution where status = ? order by create_time desc limit 100
        at org.jumpmind.db.sql.AbstractSqlTemplate.translate(AbstractSqlTemplate.java:300)
        at org.jumpmind.db.sql.JdbcSqlReadCursor.<init>(JdbcSqlReadCursor.java:120)
        at org.jumpmind.db.sql.JdbcSqlTemplate.queryForCursor(JdbcSqlTemplate.java:140)
        at org.jumpmind.db.sql.AbstractSqlTemplate.query(AbstractSqlTemplate.java:199)
        at org.jumpmind.db.sql.AbstractSqlTemplate.query(AbstractSqlTemplate.java:195)
        at org.jumpmind.db.sql.AbstractSqlTemplate.query(AbstractSqlTemplate.java:185)
        at org.jumpmind.db.sql.AbstractSqlTemplate.query(AbstractSqlTemplate.java:121)
        at org.jumpmind.metl.core.persist.ExecutionService.findExecutions(ExecutionService.java:233)
        at sun.reflect.GeneratedMethodAccessor185.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:333)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:133)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:121)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:213)
        at com.sun.proxy.$Proxy41.findExecutions(Unknown Source)
        at org.jumpmind.metl.ui.views.manage.ManageView.getBackgroundData(ManageView.java:331)
        at org.jumpmind.metl.ui.views.manage.ManageView.onBackgroundDataRefresh(ManageView.java:288)
        at org.jumpmind.metl.ui.init.BackgroundRefresherService.run(BackgroundRefresherService.java:122)
        at org.jumpmind.metl.ui.init.BackgroundRefresherService.refresh(BackgroundRefresherService.java:89)
        at org.jumpmind.metl.ui.init.BackgroundRefresherService.lambda$setPollingInterval$0(BackgroundRefresherService.java:77)
        at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

--- a/metl-core/src/main/java/org/jumpmind/metl/core/persist/ExecutionService.java
+++ b/metl-core/src/main/java/org/jumpmind/metl/core/persist/ExecutionService.java
@@ -234,7 +234,7 @@ public class ExecutionService extends AbstractService implements IExecutionServi
                 "select id, agent_id, flow_id, deployment_id, deployment_name, agent_name, "
                 + "host_name, flow_name, status, start_time, end_time, create_by, last_update_by, parameters "
                         + "from %1$s_execution " + "where " + whereClause
-                        + "order by create_time desc limit " + limit, tablePrefix),
+                        + "order by create_time desc", tablePrefix), limit,
                 new ISqlRowMapper<Execution>() {
                     public Execution mapRow(Row row) {
                         Execution e = new Execution();

